android/app/build/generated/source/buildConfig/debug/com/example/smkit_ui_flutter_plugin_example/BuildConfig.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Automatically generated file. DO NOT MODIFY
+ */
+package com.example.smkit_ui_flutter_plugin_example;
+
+public final class BuildConfig {
+  public static final boolean DEBUG = Boolean.parseBoolean("true");
+  public static final String APPLICATION_ID = "com.example.smkit_ui_flutter_plugin_example";
```

Comment:
@SencyCaspit maybe its better to use "com.sency.smkit_ui_flutter_plugin_example"